### ISSUE TITLE
Settings aren't saved after typing and closing window

### ISSUE THREAD

**Issue by Alex-K1m** [slgobinath/SafeEyes#425]
## How to reproduce

1. open settings
2. focus on any input field by clicking it (not the `+` `-` buttons)
3. change the number using keyboard
4. close the window
5. check settings again - the number you typed isn't there

## Expected

There're no OK/Cancel buttons so I expect the app to save the settings even when I'm typing and closing the window.

## Additional info

Works fine if I

- focus on another control
- switch to another tab inside the settings
- `alt+tab` to another app

before closing the window.

SafeEyes v2.1.2

<details>
<summary>System</summary>

Operating System: Manjaro Linux
KDE Plasma Version: 5.20.5
KDE Frameworks Version: 5.78.0
Qt Version: 5.15.2
Kernel Version: 5.10.15-1-MANJARO
OS Type: 64-bit
Processors: 8 × Intel® Core™ i5-8250U CPU @ 1.60GHz
Memory: 7,7 GiB of RAM
Graphics Processor: Mesa Intel® UHD Graphics 620
</details>

**Comment by slgobinath** [comment]
Yup this is a known issue with `GtkAdjustment` widget. If the value is changed using keyboard, enter key must be pressed to apply the changes. To fix this, I may have to implement some hacks for all `GtkAdjustment` widgets.

So I am planning to wait until the GTK team fix this issue (I am not sure if they aware of this problem or if they consider this as a feature or bug).
